python-sdk/docs/configurations.rst

```diff
@@ -123,3 +123,21 @@ or by updating Airflow's configuration
 
    [astro_sdk]
    auto_add_inlets_outlets = True
+
+
+Configuring the native fallback mechanism
```

Comment:
Should we update point 3 in https://astro-sdk-python.readthedocs.io/en/stable/astro/sql/operators/load_file.html#parameters-for-native-transfer too @pankajastro ?

Reply:
updated 6dccda0ad378c40578138421e5c21692fde27395